django/db/migrations/loader.py

```diff
@@ -64,6 +64,10 @@ def load_disk(self):
         self.unmigrated_apps = set()
         self.migrated_apps = set()
         for app_config in apps.get_app_configs():
+            # make it possible to skip migrations during tests
+            if self.connection and not self.connection.settings_dict.get('MIGRATE', True):
```

Comment:
Adding a

```py
# pseudo code
if settings.SETTING_NAME_HERE is False:
    return None
```

check to `migrations_module()` would make the code much testable. Or you've already tried and it didn't work for you?